src/main/java/net/neoforged/neoforge/common/extensions/IItemExtension.java

```diff
@@ -129,11 +129,11 @@ default boolean makesPiglinsNeutral(ItemStack stack, LivingEntity wearer) {
     }
 
     /**
-     * Called by CraftingManager to determine if an item is reparable.
+     * Called by CraftingManager and Grindstone to determine if an item is repairable by combining two damaged versions together.
```

Comment:
The javadoc is outdated, since this is now called in `RepairItemRecipe`. Perhaps it would be better to synchronize this javadoc with the one from `IItemStackExtension`. (Also, the typo of `reparable` is still in the `@return` tag.)